packages/tc-schema-validator/validation/index.js

```diff
@@ -82,7 +82,7 @@ const signpost = error => {
 
 const fail = () => {
 	console.error('Treecreeper schema files invalid');
-	process.exit(2);
+	// process.exit(2);
```

Comment:
Temporary code whilst @nl-ria applies the validator upgrade - forces all yamls to be treated as valid.